psi4/driver/procrouting/dft_funcs/superfuncs.py

```diff
@@ -92,6 +92,7 @@
             sup.set_name(sup.name() + '-D3M')
             superfunctional_list.append(sup)
```

Comment:
Right here add the following to get `energy('func-NL')` to work:

```
        if dashlvl == 'nl':
            # -D2 overide
            sup = superfunctionals[func](func, 1, 1, True)[0]
            sup.set_name(sup.name() + '-NL')
            superfunctional_list.append(sup)
```

Reply:
This bit is now gone - the handling of dispersion is done in `dict_builder.py`. As long as `qcdb.dashparam.get_dispersion_aliases("nl")` returns `"nl"`, the `"dispersion"` key in the dictionary of the functional will be sourced from `qcdb.dashparam.dashcoeff["nl"][func]` automatically.